airbyte-config/migrations/src/main/java/io/airbyte/config/migration/AirbyteConfigMigration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.config.migration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * This class Applies transformations to archived data related to Airbyte Configuration files.
+ * It takes an archive path as input, transforms and output it to an outputArchive path
+ */
+public class AirbyteConfigMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AirbyteConfigMigration.class);
+
+  public void transformData(String inputArchive, String outputArchive) {
```

Comment:
what do these archives represent? are they file paths?

Reply:
I don't know yet, should it be file paths or folder paths?

Do we produce one or multiple YAML files? 

We could have one YAML per type of configuration folders here:
- DESTINATION_CONNECTION          
- STANDARD_DESTINATION_DEFINITION STANDARD_SYNC                   
- STANDARD_WORKSPACE
- SOURCE_CONNECTION               
- STANDARD_SOURCE_DEFINITION      
- STANDARD_SYNC_SCHEDULE

WDYT?